core-web/libs/dotcms-models/src/lib/dot-language.model.ts

```diff
@@ -1,6 +1,6 @@
 export interface DotAddLanguage {
-    country: string;
-    countryCode: string;
+    country?: string;
```

Comment:
Why optionals?

Reply:
I order to create a new one, only the language and languageCode are required. others are optional.